default/scripting/species/common/influence.macros

```diff
@@ -18,6 +18,23 @@ BASE_INFLUENCE_COSTS
                         Species
                         Not Species name = "SP_EXOBOT"
                     ] +
+                    Statistic Count condition = And [
+                        Ship
+                        OwnedBy empire = Source.Owner
+                        (LocalCandidate.OrderedColonizePlanetID != -1)
+                    ] +
+                    Statistic Count condition = And [
+                        Building
+                        OwnedBy empire = Source.Owner
+                        Building name = [
```

Comment:
Can it use something like tags or so to not require edit this list if new species will be introduced?

Reply:
probably yes, though I didn't see any suitable tag already present, and didn't want the extra complication of adding one in the v0.5 version

Reply:
doesn't seem to work, though...